for_loop.c

```diff
@@ -0,0 +1,23 @@
+// FOR LOOP
+//C program to calculate average of any n natural numbers entered through the keyboard.
+
+#include <stdio.h>
+
+int main()
+ {
+  int n, num, i;
+  float sum=0, avg;
+  printf("How many numbers:\n");
+  scanf("%d",&n);
+  printf("Enter the natural numbers:\n");
+
+  for (i= 0; i < n; i++)
+  {
+    scanf("%d", &num);
```

Comment:
What about if user enters a number that doesn't qualify as natural number

Reply:
Sir, if the number entered by the user is a negative integer, the calculation of average will not be effected. However if user enter a number which has some decimal values, the program will not be able to calculate the average.
Should I make changes in the program so that the average is calculated even  when the number is float?